docs/modules/usage/llms/llms.md

```diff
@@ -4,16 +4,26 @@ sidebar_position: 2
 
 # 🤖 LLM Backends
 
-OpenHands can connect to many LLMs. However, the recommended models to use are GPT-4 and Claude 3.5.
+OpenHands can connect to any LLM supported by LiteLLM. However, it requires a powerful model to work.
+The following are verified by the community to work with OpenHands:
+* GPT4
+* Claude 3.5
```

Comment:
Any I missed? Just went with the "Verified" list in the UI

Reply:
I've used below as well and see them as usable:
- llama-3.1-405b / hermes-3-llama-3.1-405b
- gemini-pro-1.5 /-flash
- wizardlm-2-8x22b

Reply:
The first one I thought was not feasible with OpenHands.

For the other two, is it:
* gemini-1.5-pro / gemini-1.5-flash
* wizardlm-2-8x22b

?

Reply:
Oops... my post was for the general list, not the ollama one, sorry 😬